app/assets/stylesheets/local/utilities.scss

```diff
@@ -0,0 +1,49 @@
+// utility classes used as overrides, therefore valid use of !important
```

Comment:
The following CSS change is an alternative to using these utility classes (for the purposes of this commit);

```
.step-info {
  @include core-19;
  color: $grey-1;

+ margin-top: 45px;
+ margin-bottom: 0;

  .link-back {
    @include core-19;
    margin-right: 5px;
  }

+  & + .heading-large {
+    margin-top: 0;
+  }
}

```